Makefile

```diff
@@ -0,0 +1,18 @@
+clean_translations_temp_directory:
+	rm -rf I18N/I18N/
```

Comment:
@volodymyr-chekyrta the commands here are mostly not working, but it serves as a skeleton for us to build upon.